t2t/nlevel.py

```diff
@@ -564,7 +563,7 @@ def make_consensus_tree(cons_split, check_for_rank=True, tips=None):
     god_node = TreeNode(name=None)
     god_node.Rank = None
 
-    base = cons_split[0]
+    base = next(iter(cons_split))
```

Comment:
@wasade also here, I am not totally sure if the syntax change preserves your logic